src/telegraf.ts

```diff
@@ -92,11 +91,11 @@ export class Telegraf<C extends Context = Context> extends Composer<C> {
     return this.telegram.token
   }
 
-  /** @deprecated use `ctx.telegram.webhookReply` */
   set webhookReply(webhookReply: boolean) {
     this.telegram.webhookReply = webhookReply
   }
 
+  /** @deprecated use `ctx.telegram.webhookReply` */
```

Comment:
Changing these deprecation hints looks unrelated to the PR? (same with L95)
Also I'm not sure why one (set or get) is deprecated but not both?

Reply:
Thanks for reviews!

Yes, it's unrelated. I just fix small issues in the files I'm already editing.

Getter has to be deprecated, otherwise someone can think that the property is becoming read-only. Setter does not, write-only properties make very little sense, but might as well.